x-pack/metricbeat/module/aws/aws.go

```diff
@@ -92,6 +92,7 @@ func NewMetricSet(base mb.BaseMetricSet) (*MetricSet, error) {
 		Latency:       config.Latency,
 		AwsConfig:     &awsConfig,
 		TagsFilter:    config.TagsFilter,
+		Endpoint:      config.AWSConfig.Endpoint,
```

Comment:
Should we fail if both `Endpoint` and `AWSPartition` are provided?

Reply:
Yep, you are right, I was thinking on `endpoint` as a new setting, but it existed before.

Would it make sense to log a warning when both are used mentioning the endpoint and region that are going to be used?

Reply:
Done :) Thanks!